ouroboros-consensus-cardano-test/test/Test/Consensus/Cardano/ByronCompatibility.hs

```diff
@@ -1,13 +1,15 @@
-{-# LANGUAGE FlexibleContexts      #-}
-{-# LANGUAGE FlexibleInstances     #-}
-{-# LANGUAGE GADTs                 #-}
-{-# LANGUAGE MultiParamTypeClasses #-}
-{-# LANGUAGE RankNTypes            #-}
-{-# LANGUAGE ScopedTypeVariables   #-}
-{-# LANGUAGE StandaloneDeriving    #-}
-{-# LANGUAGE TypeApplications      #-}
-{-# LANGUAGE TypeFamilies          #-}
-{-# LANGUAGE UndecidableInstances  #-}
+{-# LANGUAGE DerivingStrategies         #-}
```

Comment:
Some of these changes (at least `DerivingStrategies`) seem unnecessary.

Reply:
reverted